File: dddnetcore/Domain/Staffs/EditingStaffDto.cs
using System;

namespace dddnetcore.Domain.Staffs
{
    public class EditingStaffDto
    {
        public string Email {get;set;}
        public string PhoneNumber {get;set;}
        public Guid? SpecializationId {get;set;}
        public long? NewAvailabilitySlotStartTime {get;set;}
        public long? NewAvailabilitySlotEndTime {get;set;}
        public Guid? ToRemoveAvailabilitySlotId {get;set;}
    }
}

File: frontend/src/app/domain/EditingStaffDto.ts
export interface EditingStaffDto {
    email: string | null;
    phoneNumber: string | null;
    specializationId: string | null;
    newAvailabilitySlotStartTime: number | null;
    newAvailabilitySlotEndTime: number | null;
    toRemoveAvailabilitySlotId: string | null;
}